buildenv

```diff
@@ -0,0 +1,37 @@
+export ZOPEN_TARBALL_URL="https://www.gnupg.org/ftp/gcrypt/pinentry/pinentry-1.2.1.tar.bz2"
+export ZOPEN_TARBALL_DEPS="make bzip2 tar curl perl automake autoconf m4 sed gettext zoslib diffutils libassuan libgpgerror"
+export ZOPEN_TYPE="TARBALL"
+export ZOPEN_COMP=CLANG
+export ZOPEN_CONFIGURE_OPTS=" --prefix=/home/haritha/zopen/prod/pinentry-1.2.1/ --enable-pinentry-tty"
```

Comment:
Use ZOPEN_EXTRA_CONFIGURE_OPTS, that way you don't need to specify prefix.

Reply:
Yes, it helped, I have modified it.